components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -29,7 +30,7 @@ object ErrorPages {
             .replace("%pageTitle%", context.getString(R.string.errorpage_title))
             .replace("%button%", context.getString(R.string.errorpage_refresh))
             .replace("%messageShort%", context.getString(errorType.titleRes))
-            .replace("%messageLong%", context.getString(errorType.messageRes))
+            .replace("%messageLong%", context.getString(errorType.messageRes, uri ?: ""))
```

Comment:
Since you've added a default value of empty string for this parameter, then you don't need to use the elvis operator to do a null-check.

That said, in my test, `Context#getString` already handles nullable values being passed to it, so you don't have to do this check.

Keeping it as just `context.getString(errorType.messageRes, uri)` will work along with your changes above.

Reply:
Thanks, that can be removed.